daffodil-tdml-processor/src/test/resources/test/tdml/tdmlQuoting.tdml

```diff
@@ -93,7 +93,7 @@
 
 	<tdml:parserTestCase name="apos_test2"
 		root="apostrophe" model="apostrophe"
-		description="Can't deal properly with apostrophe (&apos;) marks within the document data in TDML " roundTrip="true">
+		description="Shows proper handling of apostrophe (&apos;) marks within the document data in TDML " roundTrip="true">
```

Comment:
Can you rebase this change onto master and remove the apostrophy changes that are in a separate commit?

Reply:
Yes. Mistake.